src/main/java/com/chaseatucker/songr/CapitalizeController.java

```diff
@@ -0,0 +1,15 @@
+package com.chaseatucker.songr;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+public class CapitalizeController {
+  @RequestMapping(value = "/capitalize/{word}", method = RequestMethod.GET)
```

Comment:
You may be able to simplify your code only have one line to addAtribute. You may also be able use a @GetMapping, versus a @RequestMapping, which reduces the typing required. 
```
@GetMapping("/capitalize/{word}")
    public String capitalizeIt(@PathVariable String word, Model m) {
        m.addAttribute("word", word.toUpperCase());
        return "capitalize";
    }
```